dependencies.props

```diff
@@ -53,6 +53,12 @@
     <FeedTasksPackageVersion>2.1.0-prerelease-02221-02</FeedTasksPackageVersion>
   </PropertyGroup>
 
+  <!-- Publish symbol build task package -->
+  <PropertyGroup>
+    <PublishSymbolsPackage>Microsoft.SymbolUploader.Build.Task</PublishSymbolsPackage>
+    <PublishSymbolsPackageVersion>1.0.0-alpha-00001</PublishSymbolsPackageVersion>
```

Comment:
This is too tightly coupled to the version number in the buildtools repo. We need a better way.

Reply:
And we also need this version to be auto-updated.

Reply:
@mikem8361 correct me if I'm wrong but this package is manually created so there isn't any auto-updates for it.

Reply:
Isn't that even worse?  Where does this package live?